installers/agent/win/config/wrapper-agent.conf

```diff
@@ -62,8 +62,8 @@ set.AGENT_STARTUP_ARGS=-Xms128m -Xmx256m %JVM_DEBUG% %GC_LOG% "-DJAVA_SYS_MON_TE
 
 # Application parameters.  Add parameters as needed starting from 1
 wrapper.app.parameter.1=com.simontuffs.onejar.Boot
-wrapper.app.parameter.2=%GO_SERVER%
-wrapper.app.parameter.3=8153
+wrapper.app.parameter.2=-serverUrl
```

Comment:
> Either we should handle it (meaning log proper error if this happens), or have some sort of warning somewhere (in docs/release notes?) stating the same

Not sure how we can "handle it", in such a scenario, the bootstrapper will refuse to startup and print the usage onto the console.

Will make sure to submit a PR for release notes to document this behaviour.